static/src/javascripts/es6/projects/common/modules/crosswords/main.js

```diff
@@ -416,6 +417,21 @@ class Crossword extends React.Component {
         this.save();
     }
 
+    onClearSingle () {
+        const cellsInFocus = helpers.cellsForEntry(this.clueInFocus());
+
+        this.setState({
+          grid: helpers.mapGrid(this.state.grid, (cell, gridX, gridY) => {
+              if (_.some(cellsInFocus, cell => cell.x === gridX && cell.y === gridY)) {
```

Comment:
I didn't even know the array prototype had `some`. So might as well change the whole of crosswords to `Array.prototype.some` as it's used in a few places.